swagger_spec_validator/validator20.py

```diff
@@ -5,32 +5,34 @@
 
 from swagger_spec_validator.common import (SwaggerValidationError,
                                            TIMEOUT_SEC,
-                                           validate_json)
+                                           validate_json,
+                                           wrap_exception)
 
 log = logging.getLogger(__name__)
 
 
+@wrap_exception
 def validate_spec_url(spec_url):
     """Validates a Swagger 2.0 API Specification at the given URL.
 
     :param spec_url: the URL of the api-docs.
     :returns: `None` in case of success, otherwise raises an exception.
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
-    :raises: :py:class:`jsonschema.exceptions.ValidationError`
     """
     log.info('Validating %s' % spec_url)
     spec_json = json.load(urllib2.urlopen(spec_url, timeout=TIMEOUT_SEC))
     validate_spec(spec_json)
 
 
-def validate_spec(spec_json):
+def validate_spec(spec_json, spec_url=None):
```

Comment:
The python convention is to prefix unused variables with a `_` to make it clear that it's expected that this isn't used.

`_spec_url=None`